lib/common_domain_models/lib/src/ids/abgabenziel_id.dart

```diff
@@ -71,6 +71,7 @@ class AbgabezielId extends Id {
         _id = HomeworkId(contentIdString);
         break;
       default:
+        _id = Id(contentIdString);
     }
 
```

Comment:
Does this make sense? I guess you just added it so `_id` is not null, but idk if that is correct behavior.   
For now we only have implemented it for homeworks anyway. Should we not rather throw an error?

Reply:
Yes, for now, we have it only for homework. I'm going to change to throw an error 👍